pkg/cli/listbox_test.go

```diff
@@ -452,3 +452,31 @@ func TestListBox_Select_CallOnSelect(t *testing.T) {
 	w.Select(func(ListBoxState) int { return 0 })
 	verifyOnSelect(0)
 }
+
+func TestListBox_Accept_IndexCheck(t *testing.T) {
+	tests := []struct {
+		name     string
+		nItems   int
+		selected int
+	}{
+		{"index in range", 1, 0},
+		{"index exceeds left boundary", 1, -1},
+		{"index exceeds right boundary", 0, 0},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			w := NewListBox(ListBoxSpec{
+				OnAccept: func(it Items, i int) {
+					if i >= it.Len() || i < 0 {
```

Comment:
Avoid repeating the logic of the code under test.

Instead, introduce a `shouldAccept` field in the test cases, and use `!shouldAccept` as the condition.

Reply:
make sense!